package.json

```diff
@@ -11,7 +11,7 @@
     "declaration"
   ],
   "private": false,
-  "author": "philippsimon",
+  "author": "Digital UK/philippsimon",
```

Comment:
This makes it look like philippsimon is a DUK employee

Reply:
Hmm, it does rather, I don't know if there's a convention for forked npm projects..?

Reply:
changing the author to Digital UK and adding a contributors field with philippsimon?

Reply:
it will also always say in GitHub that it was forked and give a link to the original

Reply:
Looks good - will make the change.
@joelKnapp by convention would you or I resolve the convesation when this change is made?

Reply:
I'll resolve, we should turn on the require conversations to be resolved option in the project, and consider turning on the auto dismiss stale reviews

Reply:
All sounds good. I think we need to hang on until David's around again to clarify that a public fork of this repo is what's actually wanted, but then definitiely would be good to get the repo settings all twiddled with.